blazor/combobox/style.md

```diff
@@ -134,4 +134,32 @@ Use the following CSS to customize the appearance of popup element.
     font-family: emoji;
     min-height: 29px;
 }
-```
\ No newline at end of file
+```
+
+## Readonly-mode
```

Comment:
Check whether these content are repeated in this PR, since we have already included same in the API and method PR